src/transformers/models/decision_transformer/modeling_decision_transformer.py

```diff
@@ -346,21 +572,25 @@ def forward(self, hidden_states: Optional[Tuple[torch.FloatTensor]]) -> torch.Fl
         return hidden_states
 
 
-# Copied from transformers.models.gpt2.modeling_gpt2.GPT2Block with GPT2->DecisionTransformerGPT2
+DECISIONTRANSFORMERGPT2_ATTENTION_CLASSES = {
+    "eager": DecisionTransformerGPT2Attention,
+}
```

Comment:
Where is `DecisionTransformerGPT2FlashAttention2`

Reply:
Haven't added it there, but added it now here https://github.com/huggingface/transformers/pull/29226/commits/74fb9bda09067c320d3fc28845aa897cd6e8fe09. However, `DecisionTransformer` does not support flash attention yet just I had to do these modifications to make sure nothing would break with the `Copy from` statements.